tests/ci/get_previous_release_tag.py

```diff
@@ -42,7 +43,29 @@ def find_previous_release(server_version, releases):
 
     for release in releases:
         if release.version < server_version:
-            return True, release
+
+            # Check if the artifact exists on GitHub.
+            # It can be not true for a short period of time
+            # after creating a tag for a new release before uploading the packages.
+            if (
+                requests.head(
+                    CLICKHOUSE_PACKAGE_URL.format(
+                        version=release.version, type=release.type
+                    ),
+                    total=10,
+                    read=10,
+                    connect=10,
+                    backoff_factor=0.3,
+                ).status_code
+                != 404
+            ):
+                return True, release
+            else:
+                print(
```

Comment:
It should be `logging.debug` rather than `logging.info`, but never `print`, please.

Reply:
Ok, please edit in place.
I've copy-pasted this code from some other place.